<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userwindow</class>
 <widget class="QMainWindow" name="userwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>智能社区</string>
    </property>
   </widget>
   <widget class="QLabel" name="helloLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>下午好</string>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>用户</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="manBtn">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>物业管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gameBtn">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>悠闲娱乐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="videoBtn">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>本地监控</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lightBtn">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>照明系统</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>退出登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="bgLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>bgLabel</zorder>
   <zorder>label</zorder>
   <zorder>helloLabel</zorder>
   <zorder>usernameLabel</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
